ComSemApp/templates/ComSemApp/teacher/edit_released_worksheet.html

```diff
@@ -0,0 +1,93 @@
+{% extends 'ComSemApp/sidebar.html' %}
+
+{% load static %}
+
+{% block content %}
+
+<!-- page head -->
+<form method="POST" id="editWorksheetForm">
+	{% csrf_token %}
+
+	<!-- Page-Title -->
+	<div class="row">
+		<div class="col-sm-12">
+			<div class="page-title-box">
+
+				<h4 class="page-title">
+					{% if not worksheet.id %}New Worksheet{% else %}Edit Worksheet{{ worksheet.number }}{% endif %}
```

Comment:
I think `worksheet.id` will always be non-null at this point
```suggestion
					Edit Worksheet
```